ACMILabs/interactive-label
# #121 Allow lens taps on the main screen to collect the first item

Resolves #121

Visitors to SOMI are consistently trying to collect items when individual labels aren't open and move away before the error message has appeared.

Trello: https://trello.com/c/cavRQq3F/685-interactive-labels-collect-all

### Acceptance Criteria
- [x] Allow a lens tap without selecting a label to collect the first label of the playlist
- [x] Update collect text to: `To collect tap lens on reader`
- [x] Update main screen collect message to: `Select a work to view and add to your collection`

### Relevant design files
* None

### Testing instructions
1. Point your interactive label to XOS Staging (see below)
1. Start your environment: `cd development` and `docker-compose up --build`
1. Tap on a lens reader pointing at the Interactive Label without tapping on a label
1. Note the first label of the playlist is saved to your lens in XOS Staging: https://xos-stg.acmi.net.au/lens/
1. Tap on another label and collect it, noting that process continues to work
1. Test this on a Staging device in: [s__interactive-label-x86](https://dashboard.balena-cloud.com/fleets/1765718/devices)

```env
XOS_API_ENDPOINT=https://xos-stg.acmi.net.au/api/
AUTH_TOKEN=<SNIP>
# FSF
XOS_PLAYLIST_ID=136
```

### DoD
For requester to complete:
